From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Christopho <christopho@solarus-games.org>
Date: Fri, 9 Feb 2024 16:21:19 +0100
Subject: [PATCH] Set GLM_ENABLE_EXPERIMENTAL no matter the GLM version

We include some gtx headers so I think we always need this flag.
---
 cmake/modules/FindGLM.cmake | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/cmake/modules/FindGLM.cmake b/cmake/modules/FindGLM.cmake
index 624e24240..4c0d95732 100644
--- a/cmake/modules/FindGLM.cmake
+++ b/cmake/modules/FindGLM.cmake
@@ -91,10 +91,8 @@ if(GLM_FOUND)
   set(GLM_INCLUDE_DIRS ${GLM_INCLUDE_DIR})
   set(GLM_DEFINITIONS ${PC_GLM_CFLAGS_OTHER})
 
-  # enable GLM experimental extensions for GLM 0.9.9.0 to 0.9.9.3
-  if((GLM_VERSION VERSION_EQUAL 0.9.9.0 OR GLM_VERSION VERSION_GREATER 0.9.9.0) AND (GLM_VERSION VERSION_EQUAL 0.9.9.3 OR GLM_VERSION VERSION_LESS 0.9.9.3))
-    list(APPEND GLM_DEFINITIONS "-DGLM_ENABLE_EXPERIMENTAL")
-  endif()
+  # enable GLM experimental extensions
+  list(APPEND GLM_DEFINITIONS "-DGLM_ENABLE_EXPERIMENTAL")
 endif()
 
 # create an imported target for GLM
